android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt

```diff
@@ -95,11 +97,28 @@ class MainActivity : DaggerAppCompatActivity() {
         binding.drawerLayout.doOnApplyWindowInsets { _, insets, _ ->
             binding.drawerLayout.setChildInsetsWorkAround(insets)
         }
+        binding.contentContainer.doOnApplyWindowInsets { _, insets, initialState ->
+            binding.contentContainer.updatePadding(
+                left = insets.systemWindowInsetLeft + initialState.paddings.left,
+                right = insets.systemWindowInsetRight + initialState.paddings.right
+            )
+        }
         binding.toolbar.doOnApplyWindowInsets { _, insets, initialState ->
             binding.toolbar.updateLayoutParams<ConstraintLayout.LayoutParams> {
                 topMargin = insets.systemWindowInsetTop + initialState.margins.top
             }
         }
+        binding.navView.doOnApplyWindowInsets { _, insets, initialState ->
+            binding.navView.apply {
+                // On seascape mode only, nav bar is overlapped with DrawerLayout.
+                // So set left padding and reset width.
+                val leftSpace = insets.systemWindowInsetLeft + initialState.paddings.left
+                updatePadding(left = leftSpace)
+                updateLayoutParams<ViewGroup.MarginLayoutParams> {
+                    width = resources.getDimensionPixelSize(R.dimen.nav_drawer_width) + leftSpace
```

Comment:
Hi,
Thank you for helpful cooperation.
Sorry for the numerous request but 
It is not very appropriate to fix the width of drawer when applying to various screen sizes. 
How about this?
```
                updateLayoutParams<ViewGroup.LayoutParams> {
                    if (getWidth() > 0) { // Ignore when drawer is not created
                        width = measuredWidth + leftSpace
                    }
                }
```

Could you consider this 🙏 ?

Reply:
Good! Thanks for advices.